app/logic.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations

```

Comment:
This solution works, but you should try to split up logic to functions or classes more widely
![image](https://github.com/mate-academy/py-knights-fighting/assets/73850362/d9d06c73-8d50-4130-8e7f-45059a681077)